From 3e431a5bdd20c264a4636ddaa6b4d5a331020695 Mon Sep 17 00:00:00 2001
From: Ryuan Choi <ryuan.choi@samsung.com>
Date: Wed, 6 Aug 2014 09:00:13 +0900
Subject: [PATCH] build fix attempt with Gstreamer 1.4

@hotfix

r167025 exposed MPEG-TS metadata but looks missing includes on PlatformEfl.cmake
---
 Source/WebCore/PlatformEfl.cmake | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/Source/WebCore/PlatformEfl.cmake b/Source/WebCore/PlatformEfl.cmake
index 424f576..41ffc15 100644
--- a/Source/WebCore/PlatformEfl.cmake
+++ b/Source/WebCore/PlatformEfl.cmake
@@ -322,6 +322,16 @@ if (ENABLE_VIDEO)
         ${GSTREAMER_TAG_LIBRARIES}
         ${GSTREAMER_VIDEO_LIBRARIES}
     )
+
+    if (USE_GSTREAMER_MPEGTS)
+        list(APPEND WebCore_INCLUDE_DIRECTORIES
+            ${GSTREAMER_MPEGTS_INCLUDE_DIRS}
+        )
+
+        list(APPEND WebCore_LIBRARIES
+            ${GSTREAMER_MPEGTS_LIBRARIES}
+        )
+    endif ()
 endif ()
 
 if (WTF_USE_3D_GRAPHICS)
-- 
2.0.3

